src/components/fieldset/fieldset-test.stories.tsx

```diff
@@ -4,7 +4,7 @@ import Textbox from "../textbox";
 
 export default {
   title: "Fieldset/Test",
-  includeStories: ["Default"],
+  includeStories: ["Default", "OptionalFieldset"],
```

Comment:
**question** (non-blocking): was the required fieldset story meant to be included here too? No worries if not

Reply:
Yes it should be